### ISSUE TITLE
I have some questions about updating cheat(English is not my native language)

### ISSUE THREAD

**Issue by joy880828** [zbot473/SurvivHacks#16]
Hi !

First , I want to tell you two things :

1.English is not my native language,so if I write impolite or rude words in below article,please tell me and forgive me.

2. I know to ask you to answer my questions is impolite because I didn't pay you and we even have never talked or met , so if you are not willing to answer below question , I can understand and I wom't complain for it , and I apologize for my rude.

-------------------------------------------------------------------------------------------------------------------
I am very interested about how to update this cheat when it patched, so I check your 2 versions of this cheat and PowerOfUniverse's  latest version , and I found when cheat updated , these variable will change :
----------------------------------------------------------------
(Part 1)
File: init.js
y = e.e5d16b4d.exports.Qe,(in your latest version this line become y = e.e5d16b4d.exports.Je)
g = e.a508b62a.exports.Ce,(in your latest version this line become g = e.a508b62a.exports.Te )
v = e.a48f3bb2.exports.Ve,(in your latest version this line become v = e.a48f3bb2.exports.Ge)
h = e.c73dee75.exports.Ae,(in your latest version this line become h = e.c73dee75.exports.Ie)
x = e.d3da5587.exports.Ye,(in your latest version this line become x = e.d3da5587.exports.Ze)
-----------------------------------------------------------------------------------------------------
(Part 2 )
File: init.js
bullets: "De",(in your latest version this line become bullets: "Ce" )
planes: "Fe",(in your latest version this line become planes: "Le")
objectCreator: "ot",(in your latest version this line become objectCreator: "rt" )
pieTimer: "Ke",(in your latest version this line become pieTimer: "He")
main: "st",(in your latest version this line become main: "nt" )
main: "Te",(in your latest version this line become main: "Pe" )
players: "zt"(in your latest version this line become players: "kt" )
main: "Ge",(in your latest version this line become main: "Ue" )
itemf: "kt",(in your latest version this line become itemf: "vt" )
lootPool: "tt",(in your latest version this line become lootPool: "et" )

var t, i, o = e.scope[n.lootBarn.main][[pn.lootBarn.itemf]]()()
this line become ↓
var t, i, o = e.scope[n.lootBarn.main][n.lootBarn.itemf]
---------------------------------------------------------------------------------------------------
After I read code, I have already know how to update Part 1 Code, but Part 2 really confused me , I have no idea how to change those variable to update cheat.

So, if you're willing , please tell me how find those variable for Part 2 code.

And please let me repeat again , I know I didn't pay you any money , we even don't know each other, so if you're not willing to explain this to me , I can understand and I won't blame you , because I don't have right to do it.

Best regards

A surviv.io player

10.1.2018

P.S. : If possible , I also want to know how about my English ability , did I write anything that you can't understand or wrong grammar,mispell, inappropriate words used , if you find any condition above , please tell me ,too. Thanks a lot !

**Comment by alexvornoffice** [comment]
I think the surviv.io server is sending these variables so you need to check/to listen what the "server says"
Maybe that's how tit works or from surviv.io js files you can find the source page...

**Comment by zbot473** [comment]
In the code, I log the entire game scope into the console. Then, I see which variable names (Ce, Le, rt, etc...) correspond to which in the game code, and I change them to whatever they become. It is a trial and error process so it will take a bit of time.  
Btw use three ` to display code properly in comments.

**Comment by ThanksMrSotarks** [comment]
thanks for the guide, will try and practice on next patch update

**Comment by ThanksMrSotarks** [comment]
also, where is the "console" you are mentioning about

**Comment by zbot473** [comment]
> also, where is the "console" you are mentioning about

https://developers.google.com/web/tools/chrome-devtools/console/

**Comment by joy880828** [comment]
@zbot473 

Really appreciate for answering my question, my English ability is not good enough to express how much  my gratitude are, but really thanks !

And also thanks for telling me how to post code in comment, really thanks !

**Comment by Kalaborative** [comment]
@joy880828 I had a guide written up to find these variables in plain English a while back before I translated it into code, if you want I can update it and send it for you

**Comment by ThanksMrSotarks** [comment]
@Kalaborative please do, I really want to learn how

**Comment by Kalaborative** [comment]
@ThanksMrSotarks @joy880828 and for anyone else [here's the guide](https://docs.google.com/document/d/1idtE-KksJlafgGp97jggGjSGZmRAGgVEYAATAugfE2g/edit?usp=sharing). Hope it helps!

**Comment by joy880828** [comment]
@Kalaborative 

Really thanks !

Would you mind if I share this guide to a surviv.io Discord server ?

I trust many people in it will be interested in how to update cheat by themselves.

Thanks again.

**Comment by zbot473** [comment]
@joy880828 which one? _**Do not**_ share it with the main surviv.io server!

**Comment by joy880828** [comment]
@zbot473  I surely won't do that silly thing.

I mean 「HumphreyGaming YT DISCORD」 this one.

I know you from this Discord server , and I see there are many people discussing surviv.io cheat in it.

**Comment by ArmaanTornow** [comment]
@joy880828 discord invite?

**Comment by killercloudj** [comment]
@zbot473 can i get a discored invite?
